examples/gno.land/r/ursulovic/home/home.gno

```diff
@@ -0,0 +1,90 @@
+package home
+
+import (
+	"std"
+	"strings"
+
+	"gno.land/p/moul/md"
+	"gno.land/r/leon/hof"
+	"gno.land/p/demo/ownable"
+
+	"gno.land/r/ursulovic/registry"
+)
+
+var (
+	phrases    [10]string
+	aboutMe    string
+	pfp        string
+	pfpCaption string
+	owner *ownable.Ownable
+)
+
+func init() {
+	owner = ownable.NewWithAddress(registry.MainAddress())
+
+	phrases[0] = "Why did the blockchain go to therapy? It had a lot of unprocessed transactions."
+	phrases[1] = "Why don’t blockchains ever tell secrets? They use smart contracts – always open and never closed!"
+	phrases[2] = "In Gnoland, we don’t worry about bugs – they’re just features waiting to be integrated!"
+	phrases[3] = "Why did the smart contract break up with traditional finance? It was a long-term contract – no longer sustainable."
+	phrases[4] = "Why do blockchains make great comedians? They always deliver the punchline at just the right time!"
+	phrases[5] = "Why did the blockchain apply for a job? It wanted to become the most reliable ledger around!"
+	phrases[6] = "Why did the blockchain file a police report? It had a case of identity theft – someone stole its private keys!"
+	phrases[7] = "In Gnoland, our smart contracts aren’t just written in code – they’re also written in poetry!"
+	phrases[8] = "Why did the blockchain join a gym? To get a better hash rate!"
+	phrases[9] = "Why did the blockchain break up with traditional finance? It was looking for a more transparent relationship."
+
+	aboutMe = "Hi, I’m Ivan Ursulovic, a computer engineering graduate, blockchain enthusiast, and backend developer specializing in ASP.NET. I love learning new things and taking on challenges."
+	pfp = "https://i.ibb.co/W28NPkw/beograd.webp"
+	pfpCaption = "Belgrade, the city I come from"
+	hof.Register()
+}
+
+//Update
```

Comment:
If you want to write comments like this, I suggest you follow [godoc](https://go.dev/blog/godoc).

If you want to separate functions that are about "updating" things, write a comment 2 lines above the section. 
You can try using `goland` as an IDE :)